Yank/analyze.py

```diff
@@ -1408,20 +1433,20 @@ def analyze_directory(source_directory):
             calculation_type = ' of solvation'
 
     # Print energies
-    logger.info("")
-    logger.info("Free energy{}: {:16.3f} +- {:.3f} kT ({:16.3f} +- {:.3f} kcal/mol)".format(
+    logger.info('')
```

Comment:
Nothing major, I try to keep all string consistent because it makes it easier to search particular strings (otherwise I have to search both with `'` and `"` characters), and we usually want to have `"` inside a string so I picked the single quote.